src/app/(main)/account/settings/page.tsx

```
@@ -39,81 +39,99 @@ import { DialogContent, DialogTrigger } from "@/components/ui/dialog";
 export default function Settings() {
-  const clerk = useClerk();
+	const clerk = useClerk();
 
-  const { user, isSignedIn } = useUser();
-  const [linked, setLinked] = useState(false);
-  useEffect(() => {
-    setLinked(user?.publicMetadata.player != undefined);
-  }, [user, isSignedIn]);
+	const { user, isSignedIn } = useUser();
+	const [linked, setLinked] = useState(false);
+	useEffect(() => {
```

Biome (1.9.4)

[error] This hook specifies more dependencies than necessary: isSignedIn

This dependency can be removed from the list.


(lint/correctness/useExhaustiveDependencies)